procedures/foreman/branch.md.erb

```diff
@@ -36,31 +43,26 @@
   * [ ] Sync the community-templates to Foreman core, by running `script/sync_templates.sh`
   * [ ] Create a <%= @release %>-stable branch in [community-templates](https://github.com/theforeman/community-templates)
 * [ ] Add new languages that are at a [reasonable completion on Transifex](https://www.transifex.com/foreman/foreman/foreman/) to **develop**
+
+## Release Engineer
+
 * [ ] [Generate](https://github.com/ekohl/theforeman-rel-eng/blob/master/generate_gpg), [backup](https://github.com/ekohl/theforeman-rel-eng/blob/master/export_gpg_private), [sign](https://github.com/ekohl/theforeman-rel-eng/blob/master/sign_gpg) and [upload](https://github.com/ekohl/theforeman-rel-eng/blob/master/upload_gpg) a GPG key. See [GPG_Keys](https://projects.theforeman.org/projects/foreman/wiki/GPG_Keys) if needed.
 * [ ] Publish the key by [exporting](https://github.com/ekohl/theforeman-rel-eng/blob/master/export_gpg_public) the GPG key
   * [ ] Update the website's [security.md](https://github.com/theforeman/theforeman.org/blob/gh-pages/security.md) and create a file in [static/keys](https://github.com/theforeman/theforeman.org/tree/gh-pages/static/keys)
   * [ ] Create [releases/<%= release %>/RPM-GPG-KEY-foreman](https://yum.theforeman.org/releases/<%= release %>/RPM-GPG-KEY-foreman) on yum.theforeman.org
 
-# Package build systems
-
-* [ ] Create tool_belt config based on the previous release
-* [ ] Clone tags and create build targets in Koji: `bundle exec ./tools.rb koji --commit configs/foreman/<%= release %>.yaml`
-* [ ] Create mash configuration on Koji:
-  ```
-  bundle exec ./tools.rb mash-scripts configs/foreman/<%= release %>.yaml
-  scp mash_scripts/foreman/<%= release %>.0/*.mash root@koji.katello.org:/etc/mash/
-  ```
-  * [ ] Create mash scripts on Koji:
-  ```
-  ssh root@koji.katello.org
-  cd /usr/local/bin
-  # create foreman-client-mash-split-<%= release %>.py  foreman-mash-split-<%= release %>.py  foreman-rails-mash-split-<%= release %>.py based on the unversioned files
-  ```
-* [ ] Add new plugin tags to Koji plugins mash script (foreman-mash-split-plugins.py), remove old ones (keep three)
-* [ ] Add version <%= release %> to jobs in axes and/or combination filters, remove old ones (keep three)
+# Branching
+
+## Release Engineer
+
+* [ ] [tool_belt](https://github.com/theforeman/tool_belt)
+  * [ ] Create `configs/foreman/<%= release %>.yaml` based on the nightly and previous release
+  * [ ] Clone tags and create build targets in Koji: `bundle exec ./tools.rb koji --commit configs/foreman/<%= release %>.yaml`
 * [ ] Clone Debian nightly repos to <%= release %> using [copy/freight instructions](https://projects.theforeman.org/projects/foreman/wiki/Debian_Packaging#Branching-for-release)
```

Comment:
No because this **is** branching. We want the deb repos to be in sync with the git repo. Ideally one person executes this step and **right** after creates `deb/<%= release %>` based on `deb/develop` since we can't actually do it atomically. I'd say the same is true for `./tools.rb koji --commit ...` and branching `rpm/<%= release %>` based on `rpm/develop`.